packages/mobile/src/identity/reducer.ts

```diff
@@ -378,11 +534,32 @@ const completeCodeReducer = (state: State, numCompleteAttestations: number) => {
   }
 }
 
+const feelessCompleteCodeReducer = (state: State, feelessNumCompleteAttestations: number) => {
+  const { feelessAttestationCodes, feelessAcceptedAttestationCodes } = state
+  // Ensure numCompleteAttestations many codes are filled
+  for (let i = 0; i < feelessNumCompleteAttestations; i++) {
+    feelessAttestationCodes[i] = feelessAcceptedAttestationCodes[i] || {
+      code: ATTESTATION_CODE_PLACEHOLDER,
+      issuer: ATTESTATION_ISSUER_PLACEHOLDER,
+    }
+  }
```

Comment:
This is a state mutation. Two ways to avoid this would be to 1. Rewrite this as a `map` call or 2. copy `feelessAttestationCodes` before modifying it.

Reply:
Great catch. I copied this without checking it. Will correct them both.